modules/server/searchbroker/searchbroker-ui.service.tmpl

```diff
@@ -27,7 +27,7 @@ ExecStart=/usr/bin/docker run \
   -e MOLGENIS_PASSWORD='$${MOLGENIS_PASSWORD}' \
   -e FEATURE_STRATIFIER=${feature_stratifier} \
   -e FEATURE_STRATIFIER_MIN_NO_BIOBANKS=${stratifier_min_no_biobanks} \
-  -e TERMINOLOGY_SERVER_URL='""' \
+  -e TERMINOLOGY_SERVER_URL='/icd10' \
```

Comment:
Not sure if we need to write /icd10/fhir (see Zulip Screenshot)

Reply:
No because in Nginx `/icd10` is replaced with `/fhir` for the FHIR server backend.